WeddingSiteBuilder/index.html

```diff
@@ -51,7 +51,7 @@
 
             //$.ajax({
             //    type: "POST",
-            //    url: "api/Person",
+            //    url: "/api/Person",
```

Comment:
This is probably what was causing your ajax posts to fail because this works fine from index.html which is on the root directory so it goes to localhost:55889/api/Person but when you do this from a view it tries to hit localhost:55889/Views/api/Person